app/javascript/gobierto_data/webapp/components/sets/VisualizationsItem.vue

```diff
@@ -266,6 +269,8 @@ export default {
         this.saveLoader = false
       }
     },
+    //TODO: Extract to common.js
```

Comment:
but you have already extracted it to common.js, right?

Reply:
Yes!